modules/mobile/spec/support/iam_session_helper.rb

```diff
@@ -36,11 +36,13 @@ def iam_sign_in(iam_user = FactoryBot.build(:iam_user), access_token = nil)
 
   config.before :each, type: :request do
     Flipper.enable('mobile_api')
+    Flipper.enable('va_online_scheduling')
```

Comment:
Since the VAOS specs don't disable 'va_online_scheduling' again because our mobile users have a list of available services our specs will break if the VAOS specs run before ours.

Reply:
Prefer to have specs make no assumptions about the current state. Seems correct to do whatever is needed in setup, like this.